README.md

```diff
@@ -1,6 +1,306 @@
+- [Kubernetes](https://github.com/kubernetes/kubernetes): to run local K8s cluster
+- [Gluster-Kubernetes Deploy Script](https://github.com/copejon/gk-cluster-deploy): to deploy GCP K8s cluster with Gluster-Kubernetes Object Storage and CNS Object Broker
+- Kubernetes-Incubator [Service-Catalog](https://github.com/kubernetes-incubator/service-catalog): to provide Service-Catalog helm chart
+- Kubernetes [Helm](https://github.com/kubernetes/helm#install): to install Service-Catalog chart
+
+### Assumptions
+- Gk-cluster-deploy was written and tested on Fedora 25 and RHEL 7.
+- A [Google Cloud Platform](https://cloud.google.com/compute/) account.
+
+### Topology
+There are two primary systems that make up this demonstration
+They are the Broker and its colocated CNS Object store.
+It should be noted that the Broker can be implemented to run anywhere
+They fill the role of our micro-service provider
+It is only for the purpose of this demo that we decided deploy the Broker and the CNS Cluster in the same location
+
+
+A second system will be the locally running Kubernetes cluster on which with Service-Catalog is deployed
+This cluster will be our micro-server client
+Please refer to the [command flow diagram](docs/diagram/control-diag.md) for a more in depth look at these systems.
+
+## Setup
+### Step 0: Preparing environment
+- Clone [Kubernetes](https://github.com/kubernetes)
+- Clone [Service-Catalog](https://github.com/kubernetes-incubator/service-catalog)
+- Install [Google SDK](https://cloud.google.com/sdk/) and add `gcloud` to `PATH`
+You must have an active Google Cloud Platform account.
+- Configure `gcloud` user, zone, and project
+These are detected by the deployment script and required for setup.
+
+
+`# gcloud config set account <user account>`
+
+`# gcloud config set project <project>`
+
+`# gcloud config set compute/zone <zone>`
+
+
+### Step 1: Deploy Gluster-Kubernetes cluster in Google Compute Engine (GCE)
+This step sets up the **External Service Provider** portion of our topology (see [diagram](docs/diagram/control-diag.md)).
+
+To kick off deployment, run `gk-cluster-deploy/deploy/cluster/gk-up.sh`
+The script has a number of configurable variables relative to GCE Instance settings
+They can be found in `deploy/cluster/lib/config.sh`  These can be overridden inline with `gk-up.sh` or as environment variables. To run the script with out pausing run`gk-up.sh -y`.
+
+Runtime takes around 5 to 10 minutes
+Go get some coffee.
+
+Alternatively, you can manually deploy the GCE portion by following [these instuctions](//TODO) // TODO
+
+When deployment completes, commands to ssh into the master node and the URL of the CNS Object Broker will be output. **Note the URL and PORT.**
+
+### Step 2: Deploy the Service-Catalog
+This step sets up the **K8s Cluster** portion of our topology (see [diagram](docs/diagram/control-diag.md)).
+
+*In a separate terminal:*
+1. Change directories to the `service-catalog` repository.
```

Comment:
0. `hack/local-up-cluster.sh`  My command to bring up a local cluster is now:
`KUBELET_FLAGS="--fail-swap-on=false" KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O`